package/com.unity.formats.usd/Runtime/Plugins/x86_64/share/plugInfo.json.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: b65269c81f13f44c9920bf91d258b33b
+guid: 1e56bcd3fdf778349b3524cc6cc85793
```

Comment:
GUIDs are the fundamental file identifier in the asset database and can cause upgrade issues when they change but the file isn't actually changing -- any clue why all these GUIDs changed?

Reply:
Yeah I removed the whole folder when rebuilding to make sure I didn't have old dependencies hiding somewhere. So all the meta files were regenerated...
I should update the install script to remove all libs, keep the meta files and copy the fresh libs. I could add the usd dependencies to that as well, because it's all manual at the moment.

@lucillecaillaud is your WIP build/install script taking care of that at the moment?

Reply:
The new "binding project" I'm working on doesn't care of Unity... so I have no meta file and will never have. So unfortunatly, my script won't help you for that.